Content/OfficalGCCodeSupport.md

```diff
@@ -1,12 +1,12 @@
-# Official GCCode Support
+# Official GC/Code Support
```

Comment:
Where has there been a reference to "GC/Code" (with the slash)? Most common I've seen is "GCcode" (lowercase 'c')

Reply:
It's spelled this way on the login page: https://gccode.ssc-spc.gc.ca/users/sign_in